src/libraries/System.Management/src/System/Management/ManagementException.cs

```diff
@@ -721,7 +720,6 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
         {
             base.GetObjectData(info, context);
             info.AddValue("errorCode", errorCode);
-            info.AddValue("errorObject", errorObject);
```

Comment:
Consider serializing an explicit _null_ value for this field for compat. Otherwise the `ManagementException` deserialization ctor will throw in .NET Framework apps when given these payloads, as the ctor always expects an "errorObject" key to be present in the `SerializationInfo` dictionary.

Reply:
Ah, sure, this makes sense, will do this change.